notifications_utils/template.py

```diff
@@ -1,5 +1,6 @@
 import re
 import math
+import bleach
```

Comment:
should we now be using bleach instead of urllib.parse?

https://github.com/alphagov/notifications-utils/blob/master/notifications_utils/formatters.py#L25